redstone/client.py

```diff
@@ -418,6 +418,14 @@ def create(self, name, payload=None, raw_payload=None, root=False):
         elif payload is not None:
             data['resources'][0]['payload'] = base64.b64encode(payload).decode('utf-8')
 
+        # TKEK related fields
+        if signed_nonce is not None and iv is not None:
+            data['resources'][0]['signedNonce'] = signed_nonce
+            data['resources'][0]['iv'] = iv
+            data['resources'][0]['encryptionAlgorithm'] = 'RSAES_OAEP_SHA_256'
+        elif iv is not None and signed_nonce is None or iv is None and signed_nonce is not None:
+            ValueError("`signed_nonce` and `iv` both need to be specified for importing client encrypted payloads")
```

Comment:
Use single quotes (') not back ticks for the quoting inline... pretty sure backticks mean something different